addon/components/influencer-network-modal.js

```diff
@@ -1,10 +1,34 @@
 import Component from '@ember/component';
 import { computed } from '@ember/object';
 import { GLOBAL_SUPPORT_LINK } from '@upfluence/ember-upf-utils/resources/helpdesk-links';
+import { inject as service } from '@ember/service';
 
 const INFLUENCER_NETWORK_MODAL_COOKIE = 'upf_disable_influencer_modal';
 
 export default Component.extend({
+  currentUser: service(),
+  currentUserModel: null,
+
+  init() {
+    this._super();
+    this.currentUser.fetch().then((user) => {
+      this.set('currentUserModel', user);
+    });
+  },
+
+  displayInfluencerNetworkModal: computed(
+    'hasDisabledInfluencerNetworkModal',
+    'hideInfluencerNetworkModal',
+    'currentUserModel.companies.firstObject.billing_format',
+    function () {
+      return !this.hasDisabledInfluencerNetworkModal &&
+        !this.hideInfluencerNetworkModal &&
+        this.currentUserModel.companies.firstObject.billing_format != 'bracket'
+        ? true
+        : false;
```

Comment:
```suggestion
      return !this.hasDisabledInfluencerNetworkModal &&
        !this.hideInfluencerNetworkModal &&
        this.currentUserModel.companies.firstObject.billing_format != 'bracket'
```
Just return the boolean, it is the same

Reply:
Also, as on your other PR, check the presence of company / firstObject / billing_format

Reply:
If you are going to use at multiple spots (I can see another usage in the template), make a getter

Reply:
yes i use it in 2 files, let's talk about it tomorrow :)